components/CardForm/index.js

```diff
@@ -104,6 +141,10 @@ export default function CardForm({
     return `${hours}:${minutes}`;
   };
 
+  const [selectedTime, setSelectedTime] = useState(
+    existingActivityData?.time || getCurrentTime()
+  );
```

Comment:
If you implemented the suggestion above then you could do the following. 

```suggestion
  const [selectedTime, setSelectedTime] = useState(time || getCurrentTime());
```